### PR TITLE
Configurations section not files

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

* Renamed Files section to Configuration section per discussion
* Renamed sli.yaml and slo.yaml to be sli and slo
* Incorporated new distributor reference pages in 0.16.x.  They were added to 0.15.x with https://github.com/keptn/keptn.github.io/pull/1183 but apparently never got applied to 0.16.x.  These files were copied without modification so probably do not need review.

### CODE REVIEW

File: content/docs/0.16.x/reference/files/_index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
-title: Files
-description: Reference pages for Keptn configuration files
+title: Configurations
```
Comment (oleg-nenashev) [inline content/docs/0.16.x/reference/files/_index.md:2]: Or "Configuration files"?

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me!

**Comment by StackScribe** [comment]
This PR had stuff in it that should not have been there so I am closing it.  It is replaced by https://github.com/keptn/keptn.github.io/pull/1252

File: content/docs/0.16.x/reference/files/_index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/0.16.x/reference/files/_index.md:2]: Johannes wanted just "Configurations" but this is, of course, open to discussion.

**Comment by StackScribe** [comment]
Replaced by https://github.com/keptn/keptn.github.io/pull/1252

**Pull request closed without merging**